src/main/java/dev/snowdrop/release/App.java

```diff
@@ -182,9 +182,11 @@ public void listCVEs(
         final var cves = cveService.listCVEs(Optional.ofNullable(version));
         reportStatus(cves);
         if (release) {
+            final String repoName = "snowdrop/reports";
```

Comment:
Shouldn't this use the same repository (and probably be a constant) from the CveReportingService?